src/orders/queries.ts

```diff
@@ -244,19 +245,59 @@ export const useOrderFulfillData = makeQuery<
 >(orderFulfillData);
 
 const orderRefundData = gql`
+  ${fragmentMoney}
   query OrderRefundData($orderId: ID!) {
     order(id: $orderId) {
       id
       number
       total {
         gross {
-          amount
-          currency
+          ...Money
         }
       }
       totalCaptured {
-        amount
-        currency
+        ...Money
+      }
+      shippingPrice {
+        gross {
+          ...Money
+        }
+      }
+      lines {
+        id
+        productName
+        quantity
+        quantityFulfilled
+        unitPrice {
+          gross {
+            ...Money
+          }
+        }
+        thumbnail(size: 64) {
+          url
+        }
+      }
+      fulfillments {
+        id
+        status
+        fulfillmentOrder
+        lines {
+          id
+          quantity
```

Comment:
Do we have to pull all lines again? I believe those are the same ones from above? Also, would like  to see fragments since we display same data.

Reply:
We have to. Lines in fulfillments represents only those lines, which are fulfilled, refunded or canceled, while lines on order represent only unfulfilled lines, which are not available in any of fulfillment.

Reply:
In other words, none of these lines is pulled again.